pages/phases/job-analysis/index.html

```diff
@@ -5,15 +5,15 @@
 sidenav: hiring-phases
 phase: job-analysis
 title: Job analysis workshop
-intro: Conduct a multi-day job analysis workshop with SMEs to professionally develop your assessments. You'll help SMEs and hiring manager(s) identify and define the required competencies and proficiency levels for the position, as well as write two sets of structured interview questions that can assess the required proficiency levels. SMEs will use these during interview assessments to determine whether applicants meet the technical qualifications for the job.
+intro: Conduct a multi-day job analysis workshop with SMEs to professionally develop your assessments. You'll help SMEs and hiring manager(s) identify and define the required competencies and proficiency levels for the position, as well as write two assessments that can assess the required proficiency levels. SMEs will use the assessments to determine whether applicants meet the technical qualifications for the job.
```

Comment:
i think this is redundant - maybe remove the last sentence?

Reply:
Edited to the below in this commit: https://github.com/usds/SME-QA-Website/pull/243/commits/a46bcf0fd4e7a531e26a4a7235b46dbbbb3c3ec1

In a multi-day job analysis workshop with SMEs, you'll help SMEs and hiring manager(s) identify and define the required competencies and proficiency levels for the position. The group will then write two assessments that can assess the required proficiency levels. SMEs will use the assessments to determine whether applicants meet the technical qualifications for the job.